manifests/d/dimkohoncharov/Elixir/1.15.7/dimkohoncharov.Elixir.installer.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: dimkohoncharov.Elixir
+PackageVersion: 1.15.7
+InstallerType: nullsoft
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/elixir-lang/elixir/releases/download/v1.15.7/elixir-otp-26.exe

```

Comment:
This package doesn't write any data to control panel. Once it is installed, there is no way for WinGet to manage the package or uninstall it.